test/helpers/get-changed-files.test.ts

```diff
@@ -118,4 +145,13 @@ describe('getChangedFiles', () => {
 
     expect(result).toEqual(mock_data3[0]!.filename);
   });
+
+  it('should include original location of renamed files, as if rename was an addition and deletion', async () => {
+    (octokit.pulls.listFiles as unknown as Mocktokit).mockImplementation(async ({ page }) => ({
+      data: page === 1 ? mock_data4 : []
+    }));
+    const result = await getChangedFiles({ ignore_deleted: 'true' });
+
+    expect(result).toEqual([mock_data4[0]!.filename, mock_data4[1]!.filename, mock_data4[1]!.previous_filename].join(','));
```

Comment:
If you put `as const` after `mock_data4` you shouldn't need the non-null assertions